stdlib/mime/src/main/ballerina/src/mime/natives.bal

```diff
@@ -610,7 +663,7 @@ function externBase64Encode((string|byte[]|io:ReadableByteChannel) contentToBeEn
 # + return - If the given input is of type string, a decoded `string` is returned.
 #            If the given input is of type byte[], a decoded `byte[]` is returned.
 #            If the given input is of type io:ReadableByteChannel, a decoded `io:ReadableByteChannel` is returned.
-#            In case of errors, an `DecodeError` record is returned.
+#            In case of errors, an `mime:DecodeError` record is returned.
```

Comment:
```suggestion
#            In case of errors, an `mime:DecodeError` record is returned.
```
A decoded `string` if the given input is of type string, a decoded `byte[]`if the given input is of type byte[], a decoded `io:ReadableByteChannel` if the given input is of type `io:ReadableByteChannel`, or else a `mime:DecodeError` record in case of errors.